src/modules/auth/screens/InvitationScreen.js

```diff
@@ -83,7 +83,7 @@ export default function InvitationScreen(props) {
 			<FormText>
 				{t('InvitationScreen|If you want to invite the user manually, message them the registration URL below:')}
 			</FormText>
-			<InputGroup>
+			<InputGroup className='m-3'>
```

Comment:
On the printscreen it looks disproportional (the spaces between text, input and button), I would suggest `mt-2 mb-3` . Please attach an image of how it would look like (or add some of your margin, but the spaces between should be +- similar)

Reply:
@Pe5h4 New margin
![image](https://github.com/user-attachments/assets/be5800fb-dd66-44f5-85ff-173afd6e8f78)